### ISSUE TITLE
Back-to-top button stays visible when navigation tabs are sticky

### ISSUE THREAD

**Issue by mbelangergit** [squidfunk/mkdocs-material#3479]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Little cosmetic issue:
When using navigation tabs and configuring them as sticky, if I scroll down a page and then either scroll back to top or use the back-to-top button, the back-to-top button stays visible.

I have tested without `navigation.tabs.sticky` and in this case, the button will disappear when clicking on it or scrolling back to top of the page.

### Expected behaviour

I think the back-to-top button should disappear when the user gets back to the top of a page as when the sticky tabs are disabled.

### Actual behaviour

![image](https://user-images.githubusercontent.com/56379077/150194692-86f36b17-fca5-4c06-8332-1f363e4b2c63.png)

### Steps to reproduce

1. Configure a site with navigation tabs and sticky tabs
2. Scroll down a page
3. Scroll up to show the back-to-top button
4. Click the back-to-top button
5. The button stays there.

### Package versions

- Python: 3.10.0
- MkDocs: 1.2.3
- Material: 8.1.7

### Configuration

```yaml
theme:
  features:
    - navigation.tabs
    - navigation.tabs.sticky
    - navigation.top
```

### System information

- Operating system: Window 10
- Browser: Chrome 97.0.4692.71

**Comment by squidfunk** [comment]
Thanks for reporting. This is probably a regression in 8.1.7, which included a minor back-to-top button refactoring.

**Comment by squidfunk** [comment]
Fixed in 76d73b249.

**Comment by squidfunk** [comment]
Released as part of 8.1.8
